src/bundle/Editor.ts

```diff
@@ -279,7 +281,11 @@ export class EditorImpl extends SafeEventEmitter<EventMapInt> implements EditorI
                     extensions: this.#markupConfig.extensions,
                     disabledExtensions: this.#markupConfig.disabledExtensions,
                     keymaps: this.#markupConfig.keymaps,
-                    yfmLangOptions: {languageData: this.#markupConfig.languageData},
+                    yfmLangOptions: {
+                        languageData: getAutocompleteConfig({
+                            preserveEmptyRows: this.#mdOptions.preserveEmptyRows,
+                        }),
+                    },
```

Comment:
you should concat languageData config from `this.#markupConfig.languageData` and result of call `getAutocompleteConfig()` together